lib/Configuration.py

```diff
@@ -1100,9 +1100,35 @@ def FindLatestManifest(self, train=None, require_signature=False):
             else:
                 train = temp_mani.Train()
 
+        if enterprise:
+            mani_file = self.TryGetNetworkFile(url="%s/%s/LATEST-ENTERPRISE" % (self.UpdateServerMaster(), train),
+                                               reason="GetLatestManifest")
+            if mani_file is None:
+                log.debug("Could not get latest enterprise manifest file for train %s" % train)
```

Comment:
If there isnt LATEST-ENTERPRISE shouldnt it fall back to regular instead of returning?

Reply:
It should be, we are handling network errors here

Reply:
What if the file simply isnt there?

Reply:
Then it's a build server error. We don't want all enterprise systems to upgrade to latest TrueNAS version if someone accidentally removes `LATEST-ENTERPRISE` file, do we?